### ISSUE TITLE
"Tensor" object does not have backward()

### ISSUE THREAD

**Issue by M-Amrollahi** [yash-007/NLP-with-Deep-Learning#2]
I was going to test the code, but in the training part it raised an error telling that 
```
"Tensor" object does not have backward()
```

so, how can I calculate the backward before optimizing?

**Comment by MuhammadHashirBinKhalid** [comment]
It should be loss.backward().
In code, it says loss.backwards().
